tools/cli/go-whisk/whisk/namespace.go

```diff
@@ -39,6 +42,35 @@ type NamespaceService struct {
     client *Client
 }
 
+// Compare(sortable) compares namespace to sortable for the purpose of sorting.
+// REQUIRED: sortable must also be of type Namespace.
+// ***Method of type Sortable***
+func(namespace Namespace) Compare(sortable Sortable) (bool) {
+    // Sorts alphabetically
+    namespaceToCompare := sortable.(Namespace)
+    var namespaceString string
+    var compareString string
+
+    namespaceString = strings.ToLower(namespace.Name)
+    compareString = strings.ToLower(namespaceToCompare.Name)
+
+    return namespaceString < compareString
+}
+
+// ToHeaderString() returns the header for a list of namespaces
+func(namespace Namespace) ToHeaderString() string {
+    var boldString = color.New(color.Bold).SprintFunc()
+
+    return fmt.Sprintf("%s\n", boldString("namespaces"))
```

Comment:
This package should not know how the consumers of it operate. Here you are making the header bold to satisfy the consumer in function `printCommandsList()`.